jax/experimental/jax2tf/jax2tf.py

```diff
@@ -981,6 +981,7 @@ def _unexpected_primitive(p: core.Primitive, *args, **kwargs):
     "igamma_grad_a",
     "random_gamma_grad",
     "reduce_precision",
+    "reduce_xor_p",
```

Comment:
The name doesn't include `_p`:
```suggestion
    "reduce_xor",
```

Reply:
woops. that was my bad. Fixed.